src/components/BarGraph.js

```diff
@@ -1,24 +1,32 @@
 import { Graph } from "./Graph"
 
 class BarGraph {
-    constructor(id, title, plotOptions, dataOptions, modal, editable = true) {
+    constructor(parent_id, title, plotOptions, dataOptions, modal, editable = true) {
         this.uuid = Math.random().toString(36).substr(2, 9)
 
         this.modal = modal
 
-        this.container = document.getElementById(id)
-        this.container.setAttribute("type", "button")
-        this.container.setAttribute("data-modal-toggle", this.uuid)
+        this.companionDiv = document.createElement("div")
+        for (const style of ["p-4", "border-2", "border-gray-200", "rounded-lg"]){
+            this.companionDiv.classList.add(style)
+        }
+        this.companionDiv.style.width = "400px"
```

Comment:
why px and not viewport width or rem or something

Reply:
I'm pretty sure em, rem and the other relatives are just a certain multiplier or pixels so like 16 px is 1rem. so it's just a smaller number.

VW is relative to screen but I want these cards to stay the same width no matter what so flexbox can resize and make grid patterns . if I set card to 10 VW then it's like 10% so in mobile it's mega small